dev-util/phpstorm/phpstorm-2017.3.4.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop eutils vcs-snapshot
+
+DESCRIPTION="IDE for PHP built on the JetBrains IntelliJ IDEA platform"
+HOMEPAGE="https://www.jetbrains.com/phpstorm/"
+SRC_URI="https://download-cf.jetbrains.com/webide/PhpStorm-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( IDEA IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal )"
```

Comment:
Ok, I've tested it and I see a lot of bundled java libraries. While it'd be best if you unbundled as much as possible, I will be happy enough if you listed all the appropriate licenses of the bundled dependencies in `LICENSE`.

Reply:
I don't really want to invest that much time in this package. I've mostly followed the other IntelliJ ebuilds (`dev-util/{idea-ultimate,clion,pycharm-professional}` etc.) and none of them unbundle anything other than the JRE/JDK (which I did) or track the licenses of bundled libraries.

Reply:
I'm sorry but just because we missed it in the other packages doesn't mean we can accept packages without correct license. In fact, if what you say is true we should mask those packages until license issues are resolved.

Reply:
(good news is that we might find someone else to figure this out for you ;-))

Reply:
Actually, idea-ultimate seems to use `LICENSE=IDEA || ( ... )`. Any idea whether it's more correct that way than your way?

Reply:
Yes, I changed that because `IDEA` is the commercial license, so `LICENSE="IDEA || ( IDEA_Academic ... )"` doesn't make much sense.

The ebuilds for these packages are a bit messy, unfortunately. `dev-util/webstorm` has:
`LICENSE="WebStorm WebStorm_Academic WebStorm_Classroom WebStorm_OpenSource WebStorm_personal"`
(similarly for PyCharm)

I have no idea where these licenses came from. I have an academic license but I don't even know where I can read the terms...

Reply:
Thanks. Filed https://bugs.gentoo.org/652040.